packages/python-skia-pathops/build.sh

```diff
@@ -0,0 +1,82 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/fonttools/skia-pathops
+TERMUX_PKG_DESCRIPTION="Python bindings for the Skia library's Path Ops"
+TERMUX_PKG_LICENSE="BSD 3-Clause"
+TERMUX_PKG_MAINTAINER="Nguyen Khanh @nguynkhn"
+TERMUX_PKG_VERSION=0.8.0
+_SUFFIX='post1'
+TERMUX_PKG_SRCURL=https://github.com/fonttools/skia-pathops/archive/refs/tags/v${TERMUX_PKG_VERSION}.${_SUFFIX}.tar.gz
+TERMUX_PKG_SHA256=88bd5872bb96e19108ff7265cae2e1708f5e7f335b39ebfdd023940970e1d54c
+TERMUX_PKG_DEPENDS="libc++, python"
+TERMUX_PKG_AUTO_UPDATE=true
```

Comment:
Auto updates for skia-pathops would need to be disabled for now unless you wanna go ahead and write a `termux_pkg_auto_update()` in the build script to handle detecting, and updating the `post${INCREMENT}` part of the release tag.
Shouldn't be *too* hard.
Just need to get the tag with `termux_github_api_get_tag` and then split it between the version and _SUFFIX, then update either as appropriate.

Reply:
Versions with `post` suffix usually don't change the code so I think we can skip it.

Reply:
Yep that's fair.